### DISCUSSION TITLE
Apple M1 hardware cannot open the software

### DISCUSSION THREAD

**Discussion by sashabogi** [OrcaSlicer/OrcaSlicer#231] [Q&A]
Hello,

I have tried multiple times to open the newest downloaded version of the BambuStudio-SoftFever_V1.4.2_Mac_arm64.zip file and I get the error message. I don't think I'm missing anything or using a wrong version. Does anyone else have the same issue?

![Screenshot 2023-01-25 at 8 57 07 AM](https://user-images.githubusercontent.com/110927020/214584414-5af90163-0a60-4982-82eb-92191155b49d.jpg)

**Comment by erebus04** [comment]
Have you read the note on the Main page about this issue and the articel that are linked? https://www.howtogeek.com/803598/app-is-damaged-and-cant-be-opened/

**Reply by sashabogi** [reply]
Mo I didn't, but now that I did it works like a charm.

You are a life saver, I greatly appreciate your help.

Thank you

**Reply by erebus04** [reply]
You are Welcome 👍🏻
